File: src/test/resources/same/operators/Operators.cpp
#include <iostream>

void andOperator() {
    if (true && true) {
        std::cout << "";
    }
}

void orOperator() {
    if (true || true) {
        std::cout << "";
    }
}

void ternaryOperator() {
    int a, b;
    a = 10;
    b = (a == 1) ? 20 : 30;
}

void isEqualOperator() {
    if (true == true) {
        std::cout << "";
    }
}

void isNotEqualOperator() {
    if (true != true) {
        std::cout << "";
    }
}

void declareVariableOperator() {
    int x;
}

void initializeVariableOperator() {
    int x = 0;
}

void assignVariableOperator() {
    int x = 0;
    x = 1;
}

File: src/test/resources/same/operators/Operators.cs
using System;

public class Operators
{
    public void andOperator()
    {
        if (true && true)
        {
            Console.WriteLine("");
        }
    }

    public void orOperator()
    {
        if (true || true)
        {
            Console.WriteLine("");
        }
    }

    public void isEqualOperator()
    {
        if (true == true)
        {
            Console.WriteLine("");
        }
    }

    public void isNotEqualOperator()
    {
        if (true != true)
        {
            Console.WriteLine("");
        }
    }

    public void ternaryOperator()
    {
        int a, b;
        a = 10;
        b = (a == 1) ? 20 : 30;
    }

    public void declareVariableOperator()
    {
        int x;
    }

    public void initializeVariableOperator()
    {
        int x = 0;
    }

    public void assignVariableOperator()
    {
        int x = 0;
        x = 1;
    }
}